OpenSim/Simulation/Test/testInitState.cpp

```diff
@@ -146,23 +146,44 @@ void testStates(const string& modelFile)
     ASSERT(max(abs(y1-y2)) > 1e-4);
 }
 
+size_t getMedianOfCurrentRSSOverNSamples(size_t nsamples) {
+    // NOTE: getCurrentRSS() can be unreliable at evaluating the memory usage
+    // of a current process because of caching, shared memory and a litany of 
+    // other reasons including some platform specific details.
+    // Taking an average over nsamples just in case we get a flaky estimate.
+    // A stable estimate of memory usage is necessary so that we do not find
+    // cases where loading model either decreases or has no change in memory
+    // usage, which is impossible.
```

Comment:
I agree; this change does not belong in this PR.

If this PR will not pass tests no matter what without such a change, then I would prefer a separate PR into master, which then also gets merged into this PR before it gets merged itself.